### ISSUE TITLE
remove ariaRegion default

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-graphic#113]
### Subject of the issue
ARIA regions are used to describe the component type. It makes sense we do this for interactive components but it's a nuisance for an image. Only images with alt text provided should be announced by screen readers.  

Note, there is no default defined in [properties.schema](https://github.com/adaptlearning/adapt-contrib-graphic/blob/e38047d5e7ed2300f1be648b35f2cd5973431fdf/properties.schema#L10). It only exists in [course.schema](https://github.com/adaptlearning/adapt-contrib-graphic/blob/e38047d5e7ed2300f1be648b35f2cd5973431fdf/schema/course.schema.json#L27). 

### Expected behaviour
Images without alt text should be skipped by screen readers. There's no need for an ARIA region.

### Actual behaviour
ARIA region "graphic" is announced even when no image at text exists.
